go.mod

```diff
@@ -45,4 +45,7 @@ require (
 	go.uber.org/zap v1.20.0
 )
 
-replace github.com/tendermint/tendermint v0.34.14 => github.com/celestiaorg/celestia-core v0.34.14-celestia
+replace (
+	github.com/libp2p/go-libp2p-pubsub v0.5.7-0.20211029175501-5c90105738cf => github.com/celestiaorg/go-libp2p-pubsub v0.5.7-0.20220202152246-c33ecdf03b34
```

Comment:
A hack to do less work, but still have the newest pubsub features, excluding the commit that breaks dependencies for us(#306)

Reply:
Yop

Reply:
Done https://github.com/celestiaorg/celestia-node/issues/436

Reply:
Note that fixing this would indirectly require an update to IPLDv1, which we should do at some point anyway.